benches/hashmap_benchmark.rs

```diff
@@ -0,0 +1,228 @@
+// The benchmarks aim to only measure times of the operations in their names.
+// That's why all use Bencher::iter_batched which enables non-benchmarked
+// preparation before running the measured function.
+// Insert (which doesn't completely avoid updates, but makes them unlikely),
+// remove and search have benchmarks with empty values and with custom structs
+// of 42 64-bit integers.
+// (as a sidenote, the performance really differs; in the case of remove, the
+// remove function itself returns original value - the benchmark doesn't use
+// this value, but performance is significantly worse - about twice on my
+// machine - than the empty value remove; it might be interesting to see if a
+// remove function returning void would perform better, ie. if the returns
+// are optimized - omitted in this case).
+// The counts of inserted/removed/searched elements are chosen at random from
+// constant ranges in an attempt to avoid a single count performing better
+// because of specific HW features of computers the code is benchmarked with.
+
+extern crate concread;
+extern crate criterion;
+extern crate rand;
+
+use concread::hashmap::*;
+use criterion::{criterion_group, criterion_main, BatchSize, Criterion};
+use rand::{thread_rng, Rng};
+use std::mem;
+
+// ranges of counts for different benchmarks:
+const INSERT_COUNT: (usize, usize) = (120, 140);
```

Comment:
You could consider INSERT_COUNT_MIN and INSERT_COUNT_MAX as consts instead, or a struct with min/max fields? That would be a bit clearer for the code to see what's min and max. For example, SEARCH_SIZE, it's hard to see how it relates even with the comment?